scripts/run_backend_tests.py

```diff
@@ -90,16 +89,24 @@
     os.path.join(common.THIRD_PARTY_DIR, 'webencodings-0.5.1'),
 ]
 
-COVERAGE_PATH = os.path.join(
-    os.getcwd(), '..', 'oppia_tools', 'coverage-4.5.4', 'coverage')
+# Explicitly pass all current environment variables into subprocess.
+# Otherwise, there will be dll error for running coverage. (For Windows)
+SUBPROCESS_ENV = {
+    k.encode('utf-8'): v.encode('utf-8') for k, v in os.environ.items()
```

Comment:
Just noticed a new problem, this may cause `UnicodeDecodeError` when encoding the environment variables. 

In my case, I was using `oh-my-zsh` to customizes the prompt, and the theme is adding some non-ascii codes into the shell prompt via change environment variable `PS1`. In that case, the script fails immediately. The reason behind this (my understanding) is python will try to decode it with `ascii` first and then encode it if the object to encode is already a `str`. 
```
In [1]: import os

In [2]: a = os.environ['PS1']

In [3]: a.encode('utf-8')
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-3-7176b8d1174b> in <module>()
----> 1 a.encode('utf-8')

UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 23: ordinal not in range(128)

In [4]: a.decode('utf-8').encode('utf-8')
Out[4]: '(venv) %(?:%{\x1b[01;32m%}\xe2\x9e\x9c :%{\x1b[01;31m%}\xe2\x9e\x9c ) %{$fg[cyan]%}%c%{$reset_color%} $(git_prompt_info)'

In [5]: type(a)
Out[5]: str
```

So I have changed my local script to 
```
SUBPROCESS_ENV = {
    k.encode('utf-8'): v.encode('utf-8') if isinstance(v, unicode) else v for k, v in os.environ.items()
}
```

Reply:
Good catch, I've changed it to the suggested approach.